packages/block-library/src/buttons/block.json

```diff
@@ -62,7 +62,8 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
-		}
+		},
+		"contentRole": true
```

Comment:
Thinking on it some more, I'm wondering whether `contentRole: true` or `role: 'content'` is better 🤔 

Also wondering if it should be private to start with.

Reply:
> Also wondering if it should be private to start with.

Please not :( As a 3rd party developer this is sooooo needed

Reply:
> Some other roles for attributes were implemented quite a bit after role was added there, so there's always the possibility.

I'm only seeing [content and local](https://github.com/WordPress/gutenberg/blob/trunk/schemas/json/block.json#L190), and local doesn't seem relevant to the whole block. Might be best to keep it a boolean for now.

Reply:
Within supports, I quite like the use of booleans as it feels like "I'm going to switch this feature on", so in that sense `contentRole: true` looked pretty good to me.

Reply:
> I think it needs to be private to start with - I don't want us to be locked into this API yet. It feels confusing at the moment and we're unsure what exactly our end goal is. It's like we're sketching a drawing, not quite sure what the end will look like.

In most cases I would agree. However in this case I don't. We've had the concept of parent child blocks since the very beginning. Lists and list items, buttons with individual button blocks. Social links with social link items. Accordions with accordion items. For extenders it's tabs and tab items, sliders with slides etc. that concept isn't going away.

So whilst we can iterate with the UI and the underlying implementation of this. The fact that we need to mark container blocks as such remains present